### DISCUSSION TITLE
Some games may be not responding after hitting play

### DISCUSSION THREAD

**Discussion by sgamesdev** [Leo-Corporation/Gavilya#220] [Ideas]
The fix may be adding workingdirectory , and examle would be :
 Process p = new Process();
                        p.StartInfo.WorkingDirectory = "C:/Downloads/The Forest";
                        p.StartInfo.FileName = C:/Downloads/The Forest/TheForest.exe;
                        p.Start();

**Comment by lpeyr** [comment]
Thanks for reporting this issue, I'll investigate it and release a fix this month. 
I will try your fix as well.
